weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/Notes.tsx

```diff
@@ -53,6 +54,17 @@ export const Notes = ({
     }
   }, [isPreview, deepNotes]);
 
+  const neededUsers = useMemo(() => {
+    return Array.from(new Set(notes.map(r => r.wb_user_id)));
+  }, [notes]);
+  const users = useUsers(neededUsers);
+  const userMap = useMemo(() => {
+    if (users === 'load' || users === 'loading' || users === 'error') {
+      return {};
+    }
+    return Object.fromEntries(users.map(u => [u.id, u]));
```

Comment:
I think there's a lodash function that would be more readable here, something like
`_.keyBy(users, 'id');`?

Reply:
Fixed